src/myst/syntax/parser.cr

```diff
@@ -679,10 +680,10 @@ module Myst
         end
 
         return parse_postfix(call)
-      # While `parse_var_or_call` can distinguish Calls for string identifiers,
-      # its isolated scope (just the current token) means it cannot be
-      # responsible for parsing Calls from arbitrary expressions. However,
-      # since these are just postfix expressions, those can be handled here.
+        # While `parse_var_or_call` can distinguish Calls for string identifiers,
+        # its isolated scope (just the current token) means it cannot be
+        # responsible for parsing Calls from arbitrary expressions. However,
+        # since these are just postfix expressions, those can be handled here.
       when start = accept(Token::Type::LPAREN)
```

Comment:
I'm generally not too opposed to the Crystal formatter, but this comment block relates to the `when` on line 687, so its indentation should stay in line with it.

Reply:
Sorry about all the auto-formatting. I very much agree that it should be off, found out i could turn it off (which i did) shortly after committing the auto-formatted changes.